src/TLuaInterpreter.cpp

```diff
@@ -974,21 +964,20 @@ int TLuaInterpreter::loadRawFile(lua_State* L)
 // Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#getCurrentLine
 int TLuaInterpreter::getCurrentLine(lua_State* L)
 {
-    string luaSendText = "";
+    string window = "";
     if (lua_gettop(L) == 0) {
-        luaSendText = "main";
+        window = "main";
     } else {
         if (!lua_isstring(L, 1)) {
-            lua_pushstring(L, "getCurrentLine: wrong argument type");
-            lua_error(L);
-            return 1;
+            lua_pushfstring(L, "getCurrentLine: bad argument #1 type (MiniConsole name as string expected, got %s!)", luaL_typename(L, 1));
+            return lua_error(L);
         } else {
-            luaSendText = lua_tostring(L, 1);
+            window = lua_tostring(L, 1);
         }
     }
 
     Host& host = getHostFromLua(L);
-    QString line = host.mpConsole->getCurrentLine(luaSendText);
+    QString line = host.mpConsole->getCurrentLine(window);
```

Comment:
What happens if `window` is not a valid name?

Reply:
A string gets returned to the user